code/modules/client/preferences_mysql.dm

```diff
@@ -57,7 +56,7 @@
 					UI_style='[UI_style]',
 					UI_style_color='[UI_style_color]',
 					UI_style_alpha='[UI_style_alpha]',
-					be_special='[be_special]',
+					be_role='[sql_sanitize_text(list2params(be_special))]',
```

Comment:
The buttons work as url links through byond://, right? Couldn't someone use wireshark to see what they're sending, tweak the format, and ask for the role "; DROP TABLE players"? Anyways, I've added a proc to sql sanitize each entry of a list, so we shouldn't be at risk of this happening anymore.